src/SOS/Strike/hostcoreclr.cpp

```diff
@@ -377,37 +377,33 @@ static HRESULT GetHostRuntime(std::string& coreClrPath, std::string& hostRuntime
 
         // First attempt find the highest LTS version. We want to start with the LTSs
         // and only use the higher versions if it isn't installed.
-        if (!FindDotNetVersion(3, 1, hostRuntimeDirectory))
+        if (!FindDotNetVersion(5, 0, hostRuntimeDirectory))
         {
-            // Find highest 2.1 LTS version
-            if (!FindDotNetVersion(2, 1, hostRuntimeDirectory))
+            // Find highest 3.1.x LTS version
+            if (!FindDotNetVersion(3, 1, hostRuntimeDirectory))
             {
-                // Find highest 3.0.x version
-                if (!FindDotNetVersion(3, 0, hostRuntimeDirectory))
+                // Find highest 2.1.x LTS version
+                if (!FindDotNetVersion(2, 1, hostRuntimeDirectory))
                 {
-                    // Find highest 2.2.x version
-                    if (!FindDotNetVersion(2, 2, hostRuntimeDirectory))
+                    // Find highest 6.0.x version
+                    if (!FindDotNetVersion(6, 0, hostRuntimeDirectory))
```

Comment:
Is there any particular reason we don't just choose the highest runtime below a given number? Say `highestRelease = { 5, 0 } ` and ` highestDev = { 6, 0 } ` and just use that? This nesting is getting a bit deep.

Reply:
I wanted to be very specific about the major/minor versions and limit it to LTSs (except for 5.0).  And I'm lazy and don't really want to rewrite it.  

This approach does require either 5.0 or an LTS to be installed. If 2.2.x or 3.0 was installed only SOS won't use it. I guess I want to scope exactly the runtimes SOS will be running on.